musicrecs/static/style.css

```diff
@@ -0,0 +1,197 @@
+abbr, address, cite, code,
+del, dfn, em, img, ins, kbd, q, samp,
+small, strong, sub, sup, var,
+b, i,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section, summary,
+time, mark, audio, video {
+    margin:0;
+    padding:0;
+    border:0;
+    outline:0;
+    font-size:100%;
+    vertical-align:baseline;
+    background:transparent;
+}
+
+body {
+    line-height:1;
+}
+
+article,aside,details,figcaption,figure,
+footer,header,hgroup,menu,nav,section { 
+    display:block;
+}
+
+nav ul {
+    list-style:none;
+}
+
+blockquote, q {
+    quotes:none;
+}
+
+blockquote:before, blockquote:after,
+q:before, q:after {
+    content:'';
+    content:none;
+}
+
+a {
+    margin:0;
+    padding:0;
+    font-size:100%;
+    vertical-align:baseline;
+    background:transparent;
+}
+
+/* change colours to suit your needs */
+ins {
+    background-color:#ff9;
+    color:#000;
+    text-decoration:none;
+}
+
+/* change colours to suit your needs */
+mark {
+    background-color:#ff9;
+    color:#000; 
+    font-style:italic;
+    font-weight:bold;
+}
+
+del {
+    text-decoration: line-through;
+}
+
+abbr[title], dfn[title] {
+    border-bottom:1px dotted;
+    cursor:help;
+}
+
+table {
+    border-collapse:collapse;
+    border-spacing:0;
+}
+
+/* change border colour to suit your needs */
+hr {
+    display:block;
+    height:1px;
+    border:0;   
+    border-top:1px solid #cccccc;
+    margin:1em 0;
+    padding:0;
+}
+
+input, select {
+    vertical-align:middle;
+}
+
+/*----------------------------------------------------------------------*/
+
+/* Custom Musicrecs Content */
+
+/* Broad Styles */
+h2 {
+    margin-bottom: 20px;
+}
+
+/* Class Styles */
+.btn_main {
+    white-space: normal;
+}
+
+.div_main {
+    padding: 15px;
+    margin-top: 10px;
+    margin-bottom: 10px;
+}
+
+.div_round {
+    padding: 10px;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    margin-left: 10px;
+    margin-right: 10px;
+    display: inline-block;
+}
+
+.div_round_content {
+    padding: 15px;
+}
+
+.p_index_phase {
+    margin-left: 20px;
+}
+
+.p_index_about {
+    margin-left: 10px;
+}
+
+.p_index_faq_q {
+    margin-left: 10px;
+    font-weight: bold;
+}
+
+.p_index_faq_a {
+    margin-left: 20px;
+    font-style: italic;
+}
+
+/* ID Styles (organized by page) */
+
+/* PAGE base.html */
+#base_content {
+    margin-top: 20px;
+    margin-bottom: 20px;
+}
+
+#loading {
+    display: none;
+    cursor: wait;
+}
+
+#loading div {
+    margin-left: 50%;
+    margin-top: 20px;
+    width: 35px;
+    height: 35px;
+    background: url(/static/loadingimage.gif) no-repeat;
+}
+
+/* PAGE index.html */
+
+/* PAGE round/base.html */
+#roundLinkTextbox {
+    width: 100%;
+}
+
+/* PAGE round/listen_phase.html */
+
+/* ERROR - This is not currently functioning, but it seemed to
+have the same issue in the html code */
```

Comment:
What wasn't working before?

Reply:
That section of code was flagged as an error for me before I touched it. For scaling the album art. I think the current issue in css is because I'm using {{ }} within another set of { }. I think that is probably just a matter of finding the appropriate css symbol to use.

Reply:
when you say flagged do mean a warning by vscode? I don't see any problems visually when I tested this out